pkg/client/didexchange/client_test.go

```diff
@@ -140,7 +140,7 @@ func TestClient_CreateInvitation(t *testing.T) {
 				didexchange.DIDExchange: svc,
 				route.Coordination:      &mockroute.MockRouteSvc{},
 			},
-			KMSValue:             &mockkms.CloseableKMS{CreateEncryptionKeyValue: "sample-key"},
+			LegacyKMSValue:       &mockkms.CloseableKMS{CreateEncryptionKeyValue: "sample-key"},
```

Comment:
What's the `LegacyKMS` change?

Reply:
it's bug in unit-test. put legacykms value in wrong field